src/java/org/apache/cassandra/hints/HintsStore.java

```diff
@@ -61,6 +61,7 @@ final class HintsStore
     private final Deque<HintsDescriptor> dispatchDequeue;
     private final Queue<HintsDescriptor> corruptedFiles;
     private final Map<HintsDescriptor, Long> hintsExpirations;
+    private final Map<String, Long> hintsSizes;
```

Comment:
Why not `HintsDescriptor` as key?

Reply:
It was like that before, then I changed it to String. Is not this just better? I don't mind to revert this.